src/blocks.js

```diff
@@ -2253,11 +2257,15 @@ SyntaxElementMorph.prototype.showBubble = function (value, exportPic, target) {
         Math.max(this.rounding - 2, 6),
         0
     );
-    bubble.popUp(
-        wrrld,
-        pos,
-        isClickable
-    );
+    if (!forHelpScreen) {
```

Comment:
```suggestion
    if (forHelpScreen) { 
        return bubble;
   }
   bubble.popUp(
       wrrld,
       pos,
       isClickable
   );
```

IMO, this is a bit more clear as to when the function should return early.